student-store-ui/src/components/About/About.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
```

Comment:
As discussed in class, this line isn't needed anymore on newer version of React (namely v17+)